iib/web/exception_handlers.py

```diff
@@ -0,0 +1,37 @@
+import functools
+
+import flask
+from kombu.exceptions import OperationalError
+
+from iib.exceptions import IIBError
+from iib.web import messaging
+
+
+def catch_rabbit(request):
```

Comment:
Can we rename this to `catch_rabbitmq_error` or something better please?

Reply:
sure, no more catching rabbits it is ;)